lib/mesa-lib/DEPENDS

```diff
@@ -14,6 +14,7 @@ depends presentproto
 depends Mako
 depends nettle
 
+optional_depends wayland  "" "" "This is needed by various components of Plasma"
```

Comment:
Your right Ratler. Couldn't we get by with something like this;

—with-egl-platforms=x11,wayland,drm